easybuild/easyconfigs/p/pymatgen/pymatgen-4.1.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '4.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('spglib-python', '1.9.4.2', versionsuffix),
```

Comment:
@migueldiascosta how sure are you about that? Do you have a reference?

This looks like pure Python to me?

```
[12:50:01] vsc40023@node2636:~ $ ml spglib-python
[12:50:08] vsc40023@node2636:~ $ ls $EBROOTSPGLIBMINPYTHON
easybuild  lib
[12:50:18] vsc40023@node2636:~ $ ls $EBROOTSPGLIBMINPYTHON/lib
python2.7
[12:50:20] vsc40023@node2636:~ $ ls $EBROOTSPGLIBMINPYTHON/lib/python2.7/site-packages
spglib	spglib-1.9.4.2-py2.7.egg-info
[12:50:24] vsc40023@node2636:~ $ ls $EBROOTSPGLIBMINPYTHON/lib/python2.7/site-packages/spglib
__init__.py  __init__.pyc  spglib.py  spglib.pyc  _spglib.so
```